packages/adapter-vercel/files/entry.js

```diff
@@ -1,34 +1,43 @@
-import { getRawBody } from '@sveltejs/kit/node';
-
 // TODO hardcoding the relative location makes this brittle
 import { init, render } from '../output/server/app.js';
+import { assets } from './assets.js';
 
 init();
 
-export default async (req, res) => {
-	const { pathname, searchParams } = new URL(req.url || '', 'http://localhost');
-
-	let body;
-
-	try {
-		body = await getRawBody(req);
-	} catch (err) {
-		res.statusCode = err.status || 400;
-		return res.end(err.reason || 'Invalid request body');
+const getResult = (body, options) => ({
+	promise: Promise.resolve(),
+	waitUntil: Promise.resolve(),
+	response: new Response(body, options)
+});
+
+// eslint-disable-next-line
+_ENTRIES = {
```

Comment:
a few comments / links to vercel docs would be useful here. I don't know what `_ENTRIES`/`middleware_kit`, 'x-middleware-next', `waitUntil`, etc. are

Reply:
the docs aren't public yet, i've been sneaking a peek at some in-development stuff. agree that the API is very cryptic in its current form

Reply:
> the docs aren't public yet

Indeed! They are available [here](https://docs-git-file-system-api-docs.vercel.sh/docs/file-system-api) at the moment. We're still working on adding details about the things mentioned above, however. And the API will evolve to be simpler over the coming months — in tandem with Next.js' output. Regardless, the API currently shown in the PR will be retained forever for backward-compatibility.